source/examples/lyricsDesc/lyricsDesc-sample284.txt

```diff
@@ -0,0 +1,17 @@
+<staff n="1">
+   <layer n="1">
+      <note dur="8" dots="1" oct="3" pname="c" />
+      <note dur="16" oct="3" pname="e" accid.ges="f" />
+      <note dur="8" dots="1" oct="3" pname="g" />
+      <note dur="16" oct="3" pname="g" />
+      <note dur="4" oct="3" pname="g" />
+      <note dur="4" oct="3" pname="b" accid.ges="f" />
+   </layer>
+</staff>
+<!-- later -->
+<div>
+   <lg staff="1">
+      <l rhythm="8.,16,8.,16,8,8,4"><syl>Or</syl> <syl>let</syl> <syl con="d">Au</syl><syl>tumn</syl> <syl>fall</syl> <syl>on</syl> <syl>me</syl></l>
```

Comment:
This rhythm is not correct, please look up the original on IMSLP. Secondly I think it would be better to have a second example to explain the `@rhythm` attribute.

Reply:
Yes, we know. But also the former rhythm is not "correct"  as some prints on IMSLP show. We tried to integrate `@rhythm` into the example to avoid having a second one. But maybe we have to split them up.

Reply:
Since the text above the example mentions the rhythm attribute, I think it makes sense to have it included as one use case in that same example.

File naming is (for now) in line with general file naming. That doesn't mean I think the file naming is good as it is, but that's a secondary issue for another day ;)